.travis.yml

```diff
@@ -7,6 +7,13 @@ cache: bundler
 # @see http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
 
+addons:
```

Comment:
This is a fix for the spec tests but the dependency was not introduced in this PR.